app/_plugins/drops/plugins/hub_examples.rb

```diff
@@ -39,6 +46,18 @@ def kubernetes
         def plugin_name
           @plugin_name ||= @example.fetch('name')
         end
+
+        def auth
+          @auth ||= Examples::AuthPlugins::Base.make_for(plugin_name)
+        end
+
+        def auth_fields
+          auth.fields
+        end
+
+        def auth_next_steps_url
+          auth.next_steps_url
+        end
```

Comment:
I couldn't figure out how to make Liquid accept `Example.auth` so that I could call `fields` and `next_steps_url` in the view so I added these aliases

Reply:
that's probably because you are using classes and not instances :)
Objects used in liquid need to respond to `#to_liquid`. `Liquid::Drop` instances [respond to it](https://github.com/Shopify/liquid/blob/v4.0.4/lib/liquid/drop.rb#L49-L51) but the class doesn't.